icu4c/source/test/fuzzer/fuzzer_utils.h

```diff
@@ -30,11 +30,11 @@ std::mt19937_64 CreateRng(const uint8_t* data, size_t size) {
   return rng;
 }
 
-const icu::Locale& GetRandomLocale(std::mt19937_64* rng) {
+const icu::Locale& GetRandomLocale(uint16_t rnd) {
   int32_t num_locales = 0;
   const icu::Locale* locales = icu::Locale::getAvailableLocales(num_locales);
   assert(num_locales > 0);
-  return locales[(*rng)() % num_locales];
+  return locales[rnd % num_locales];
 }
 
 icu::UnicodeString UnicodeStringFromUtf8(const uint8_t* data, size_t size) {
```

Comment:
(The stupid GitHub review tool won't let me leave a comment on UnicodeStringFromUtf32(), this is as close as I could put it...)

uchars[i] = std::min(uchars[i], UCHAR_MAX_VALUE); is going to leave most of the values stuck at UCHAR_MAX_VALUE. Maybe mod instead of min(). Is the function even still used anywhere?

Reply:
Neither UnicodeStringFromUtf8() nor UnicodeStringFromUtf32() is used anywhere once this PR is merged.

Removed these methods and cleaned up the header file.